content/install-julia.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Installing Julia in Pop!_OS
+description: Install the Julia programming language in Pop!_OS.
+keywords:
+  - Julia
+  - Programming language
+  - Data analysis
+  - Computational science
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+## What is Julia?
+
+Julia is a high-level programming language popularly used for numerical analysis and computational science. More information can be found [on Julia's site](https://julialang.org/). Julia can be tricky to install for some new users.
+
+## Installing Julia in Pop!_OS
+
+In Pop!_OS, Julia is simply installed by running `sudo apt install julia`. Pop!\_OS will also track the latest Julia releases.
+
+```bash
+sudo apt install julia
```

Comment:
It's also installable via Pop Shop, think that's worth a mention?

Reply:
Yea definitely, I'll add that.